util/replica.go

```diff
@@ -48,32 +48,19 @@ type ReplicationInfo struct {
 
 func GetReplicationInfo(target string) (*ReplicationInfo, error) {
 	conn, err := sql.Open("postgres", target)
-
 	if err != nil {
 		log.Errorf("Could not connect to: %s", target)
 		return nil, err
 	}
-
 	defer conn.Close()
 
 	// Get PG version
-	var version int
-
-	rows, err := conn.Query("SELECT current_setting('server_version_num')")
-
-	if err != nil {
+	version := 0
```

Comment:
For the purpose of being explicit about types related to queries, I'd prefer that it stay as `var version int`.